wizeline/wize-q-remix
# feat: Migrate Question Detail Page - Re-assign department

#### What does this PR do?
Migrates Question detail Page to wize q remix
implements the assign department feature with its controller

#### Where should the reviewer start?
components/Modals/AssignAnswerModal/AssignAnswerModal.jsx
routes/questions/$questionId.jsx


#### How should this be manually tested?
1. Log in as an admin to wizeline questions
2. click on any question to enter de question detail modal
3. click on re assign question and select any department, the question should be re assigned to the selected department


#### What are the relevant tickets?
Closes #37 

#### Screenshots
<img width="1014" alt="Screen Shot 2022-11-08 at 3 05 58 p m" src="https://user-images.githubusercontent.com/45604471/200675458-32af736b-ae79-45a9-9b34-e0a4423fbf42.png">
